android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaView.java

```diff
@@ -45,12 +46,12 @@ private EdgeInsets getSafeAreaInsets() {
       int statusBarHeight = 0;
       int resourceId = getResources().getIdentifier("status_bar_height", "dimen", "android");
       if (resourceId > 0) {
-        statusBarHeight = getResources().getDimensionPixelSize(resourceId);
+        statusBarHeight = Math.round(PixelUtil.toDIPFromPixel(getResources().getDimensionPixelSize(resourceId)));
```

Comment:
Is rounding necessary? If it's actually not a round value, wouldn't it cause incorrect height?

Reply:
You are right! Changed.